supertokens_python/recipe/session/api/implementation.py

```diff
@@ -40,6 +43,11 @@ class APIImplementation(APIInterface):
     async def refresh_post(
         self, api_options: APIOptions, user_context: Dict[str, Any]
     ) -> SessionContainer:
+
+        clear_session_cookies_from_older_cookie_domain(
```

Comment:
why is this happening here, but the fix for https://github.com/supertokens/supertokens-node/pull/813 is happening in the refresh_session_in_request function?

Reply:
Moved it inside the `refresh_session_in_request` function.